modules/scramble/clusteranalysis/main.nf

```diff
@@ -0,0 +1,51 @@
+process SCRAMBLE_CLUSTERANALYSIS {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::scramble=1.0.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/scramble:1.0.1--h779adbc_1':
+        'quay.io/biocontainers/scramble:1.0.1--h779adbc_1' }"
+
+    input:
+    tuple val(meta), path(clusters)
+    path fasta
+    path mei_ref
+
+    output:
+    tuple val(meta), path("*_MEIs.txt")                 , optional:true, emit: meis_tab
+    tuple val(meta), path("*_PredictedDeletions.txt")   , optional:true, emit: dels_tab
+    tuple val(meta), path("*.vcf")                      , optional:true, emit: vcf
+    path "versions.yml"                                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def VERSION = '1.0.1' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+
+    def blastdb = args.contains("--eval-dels") ? "makeblastdb -in ${fasta} -parse_seqids -title ${fasta} -dbtype nucl -out ${fasta}" : ""
+    def reference = fasta ? "--ref `pwd`/${fasta}" : ""
+    def mei_reference = mei_ref ? "`pwd`/${mei_ref}" : "/usr/local/share/scramble/resources/MEI_consensus_seqs.fa"
```

Comment:
The path in the last part seems a bit too hard-coded for my taste.
Can you explain a bit?

Reply:
The tool itself is a bit tricky and doesn't have a lot of documentation, so most of this stuff I have to find out by experimenting and looking at other people's code. So that probably explains the weird code, sorry for that. 

That path is a default file that came with the container and is located at that path exactly (that's why this path is hard-coded)

Reply:
And also the reason why the conda tests, fail

Reply:
OK, can you add in a comment that the hard-coded path correspond to a file in the container?

Reply:
Done :)